src/main/java/de/tum/in/www1/artemis/web/rest/dto/LectureUnitsDTO.java

```diff
@@ -0,0 +1,48 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import org.springframework.web.multipart.MultipartFile;
+
+import de.tum.in.www1.artemis.domain.Attachment;
+import de.tum.in.www1.artemis.domain.lecture.AttachmentUnit;
+
+public class LectureUnitsDTO {
```

Comment:
JavaDoc + could be rewritten using a java record

Reply:
added JavaDoc and changed to java record